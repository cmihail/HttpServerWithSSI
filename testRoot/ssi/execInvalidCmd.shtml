<html>
<head>
	<title>SSI Exec</title>
</head>

<body>
<!--#exec cmd="ls invalidFolder" -->
</body>
</html>
