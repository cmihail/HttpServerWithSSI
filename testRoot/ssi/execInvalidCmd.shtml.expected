<html>
<head>
	<title>SSI Exec</title>
</head>

<body>
ls: cannot access invalidFolder: No such file or directory

</body>
</html>
